tests_perf/tests/data_generator.py

```diff
@@ -150,3 +150,11 @@ def rand_array(self):
         """Array of random floats"""
         with self.set_seed():
             return np.array(randn(self.size))
+
+
+class FloatSeriesIndexGenerator(FloatSeriesGenerator):
+
+    def generate(self):
+        ind = StringSeriesGenerator(self.size).generate()
+        d = FloatSeriesGenerator(self.size).generate()
+        return pd.Series(data = d, index = ind)
```

Comment:
Please fix according to PEP8: "Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter"